File: mario.c
#include <cs50.h> 
#include <stdio.h> 
int main(void)
{
    int n;
    int m;
    do
    {
        n = get_int("Size: ");
        //m = n - 1;
    }
    while (n < -1 || n > 8);
    for (int i = 0; i < n; i++)
    { 
        for (int j = n - i; j > 1; j--)
        {
            printf(" ");
        }
        for (int hashes = 0; hashes < i ; hashes++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: mario.py
from cs50 import get_int

def main ():
    while True:
        height = get_int("Height: ")
        if height > 0 and height <= 8:
            break
    
    for i in range (1, height + 1):
        for j in range (height - i):
            print (" ", end = "")
        for k in range(i):
            print ("#", end = "")
        print("")
        
if __name__ == "__main__":
    main()
